spec/controllers/users_controller_spec.rb

```diff
@@ -2,9 +2,9 @@
 
 RSpec.describe UsersController, type: :controller do
 
-  describe "GET #create" do
+  describe "POST #create" do
     it "returns http success" do
-      get :create
+      post :create, user: { token: "1234abcd" }
```

Comment:
That's because you used `resources :users, only: [:create]` in your routes file. The [default mapping](http://guides.rubyonrails.org/routing.html#crud-verbs-and-actions) for `create` action is a matcher for POST requests on /users.